src/signals/incident-management/containers/IncidentDetail/IncidentDetail.test.js

```diff
@@ -358,7 +417,9 @@ describe('signals/incident-management/containers/IncidentDetail', () => {
 
     await screen.findByTestId('incident-detail')
 
-    const deleteButton = screen.getByTitle(/bijlage verwijderen/i)
+    const deleteButton = screen.getByTestId(
```

Comment:
strange selector, consider something more readable

Reply:
See above, did it (partially)